### PR TITLE
[FLINK-24940] [docs] correct usage about how to create Hive catalog via Flink SQL CLI

### PR DESCRIPTION
**Pull request by luoyuxia**
## What is the purpose of the change

* As it's not supported to use yaml in SQL CLI, the documatation about how to create hive catalog in SQL CLI is out of date.
The pr is to correct the usage.
*

## Brief change log
  - *modify the hive_catalog.md*

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): no
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`:  no
  - The serializers: no
  - The runtime per-record code paths (performance sensitive):no
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn, ZooKeeper:  no
  - The S3 file system connector: no

## Documentation

  - Does this pull request introduce a new feature? no

### CODE REVIEW

File: docs/content.zh/docs/connectors/table/hive/hive_catalog.md [hunk 1]
```diff
@@ -180,11 +175,12 @@ john,21
 ```
 
 
-#### step 4: start SQL Client, and create a Kafka table with Flink SQL DDL
+#### step 4: create a Kafka table with Flink SQL DDL
 
-Start Flink SQL Client, create a simple Kafka 2.3.0 table via DDL, and verify its schema.
+Create a simple Kafka 2.3.0 table with Flink SQL DDL, and verify its schema.
```
Comment (MartijnVisser) [inline docs/content.zh/docs/connectors/table/hive/hive_catalog.md:180]: I think it would be better to remove the mentioned Kafka version

Comment (MartijnVisser) [reply docs/content.zh/docs/connectors/table/hive/hive_catalog.md:180]: I mean removing the mentioned version elsewhere on the page

**Review by MartijnVisser — APPROVED** [review]
Thanks a lot for the PR. It would be great if you could also remove the other mentions of the version number.

**Comment by MartijnVisser** [comment]
@AHeise Please merge

**Comment by luoyuxia** [comment]
@MartijnVisser Could you please help merge it ?

**Comment by MartijnVisser** [comment]
@luoyuxia Yes, let me rebase and squash commits to verify that it will still work and then I'll merge it. Apologies for the delay!

**Comment by MartijnVisser** [comment]
Build passed in https://dev.azure.com/apache-flink/apache-flink/_build/results?buildId=34343&view=results

**Pull request merged**